src/Umbraco.Web/umbraco.presentation/umbraco/dialogs/exportDocumenttype.aspx.cs

```diff
@@ -30,13 +33,32 @@ private void Page_Load(object sender, System.EventArgs e)
 			if (documentTypeId > 0) 
 			{
 				cms.businesslogic.web.DocumentType dt = new cms.businesslogic.web.DocumentType(documentTypeId);
-				if (dt != null) 
+				if (dt != null)

```

Comment:
This can never be null - you are creating a new object therefore it's impossible to be null.